project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Error/error.html.twig

```diff
@@ -39,6 +39,10 @@
                     <p>
                         {{ 'Please excuse this error, we are working on fixing it. If needed, contact us on %mail% or by phone %phone%.'|trans({ '%mail%': getShopInfoEmail(), '%phone%': getShopInfoPhoneNumber() }) }}
                     </p>
+
+                    <p>
+                        {{ 'Error ID: <span id="js-error-id">{{ERROR_ID}}</span>' }}
```

Comment:
I think this should be just
```
                        Error ID: <span id="js-error-id">{{ERROR_ID}}</span>
```
without enclosing it.

Reply:
That is not so simple, but still a good point 👍